library/kotlin/test/io/envoyproxy/envoymobile/grpc/BUILD

```diff
@@ -0,0 +1,13 @@
+licenses(["notice"])  # Apache 2
+
+load("//bazel:kotlin_test.bzl", "envoy_mobile_kt_test")
+
+envoy_mobile_kt_test(
+    name = "darwin_grpc_codec_test",
```

Comment:
This should probably not say darwin since it's running on linux

Reply:
oops -- I can make a change in another PR